db/schema.rb

```diff
@@ -1046,9 +1046,9 @@
   add_foreign_key "modern_campaign_gifts", "campaign_gifts"
   add_foreign_key "object_event_hook_configs", "nonprofits"
   add_foreign_key "offline_transaction_charges", "payments"
-  add_foreign_key "offline_transaction_refunds", "payments"
   add_foreign_key "recurrences", "recurring_donations"
   add_foreign_key "recurrences", "supporters"
+  add_foreign_key "offline_transaction_refunds", "payments"
```

Comment:
These should be alphabetical order: that's how the schema is created by Rails; if we don't the next time a migration is run, the schema will get reorganized and the changes will be a little confusing.